base_custom_attribute/models/attribute_attribute.py

```diff
@@ -96,18 +120,29 @@ def _build_attribute_field(self, subgroup):
         field = etree.SubElement(subgroup, "field", **kwargs)
         setup_modifiers(field)
 
-    def _build_attribute_view(self):
-        """Return a main_group etree element made of sub_groups for each
-        attribute_group."""
-        main_group = etree.Element("group", name="attributes_group", col="4")
+    def _build_attribute_view(self, native_efields):
+        """Return a main_att_group etree element including all the model's Attributes
+        related fields distributed in sub_groups for each Attribute's group."""
+        context_include = self._context.get("include_native_attribute")
+        main_att_group = etree.Element("group", name="attributes_group", col="4")
         groups = []
 
         for attribute in self:
             att_group = attribute.attribute_group_id
             att_group_name = att_group.name.capitalize()
+            if not att_group:
```

Comment:
Useless the field is required : https://github.com/akretion/odoo-pim/blob/b4310be93be9777d5fa9b66da66d8fdefa5d2358/base_custom_attribute/models/attribute_attribute.py#L265